HLTrigger/JetMET/plugins/HLTScoutingPFProducerV2.cc

```diff
@@ -0,0 +1,299 @@
+// -*- C++ -*-
+//
+// Package:    HLTrigger/JetMET
+// Class:      HLTScoutingPFProducerV2
+//
+/**\class HLTScoutingPFProducerV2 HLTScoutingPFProducerV2.cc HLTrigger/JetMET/plugins/HLTScoutingPFProducerV2.cc
+
+Description: Producer for ScoutingPFJets from reco::PFJet objects, ScoutingVertexs from reco::Vertexs and ScoutingParticles from reco::PFCandidates
+
+*/
+//
+// Original Author:  Dustin James Anderson
+//         Created:  Fri, 12 Jun 2015 15:49:20 GMT
+//
+//
+
+// system include files
+#include <memory>
+
+// user include files
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/global/EDProducer.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+#include "DataFormats/JetReco/interface/PFJet.h"
+#include "DataFormats/METReco/interface/PFMET.h"
+#include "DataFormats/METReco/interface/PFMETCollection.h"
+#include "DataFormats/ParticleFlowCandidate/interface/PFCandidateFwd.h"
+#include "DataFormats/ParticleFlowCandidate/interface/PFCandidate.h"
+#include "DataFormats/VertexReco/interface/Vertex.h"
+#include "DataFormats/VertexReco/interface/VertexFwd.h"
+#include "DataFormats/BTauReco/interface/JetTag.h"
+
+#include "DataFormats/Scouting/interface/Run3ScoutingPFJet.h"
+#include "DataFormats/Scouting/interface/Run3ScoutingParticleV2.h"
+#include "DataFormats/Scouting/interface/Run3ScoutingVertex.h"
+
+#include "DataFormats/Math/interface/deltaR.h"
+
+#include "DataFormats/Math/interface/libminifloat.h"
+
+#include "RecoBTag/FeatureTools/interface/TrackInfoBuilder.h"
+#include "RecoBTag/FeatureTools/interface/deep_helpers.h"
```

Comment:
You are correct that the TrackInfoBuilder.h is not necessary. However deep_helpers.h is needed to compile [these two lines](https://github.com/cms-sw/cmssw/pull/36840/files#diff-87d3bd14df755ce99831c3fbcd4d87245fd0cb4c11025301c20e4c4e8b6d2e2dR183-R184). Without it I receive the following when I do `scram b`

```
/afs/cern.ch/work/a/adlintul/public/scouting/david/pr-to-cmssw/CMSSW_12_3_0_pre4/src/HLTrigger/JetMET/plugins/HLTScoutingPFProducerV2.cc:183:19:error: 'btagbtvdeep' has not been declared
  183 |             trk ? btagbtvdeep::lost_inner_hits_from_pfcand(*reco_cand) : 0,
      |                   ^~~~~~~~~~~
/afs/cern.ch/work/a/adlintul/public/scouting/david/pr-to-cmssw/CMSSW_12_3_0_pre4/src/HLTrigger/JetMET/plugins/HLTScoutingPFProducerV2.cc:184:19: error: 'btagbtvdeep' has not been declared
  184 |             trk ? btagbtvdeep::quality_from_pfcand(*reco_cand) : 0,
      |  
```

Please let me know if there is a better way to avoid the compilation error.

Reply:
Okay, thanks. I had missed that.

Reply:
Fixed [here](https://github.com/cms-sw/cmssw/pull/36840/commits/f933d59922918b5c0c43b53e01f30c1c9233702c#diff-87d3bd14df755ce99831c3fbcd4d87245fd0cb4c11025301c20e4c4e8b6d2e2dL39).